docs/getting_started/install.rst

```diff
@@ -62,12 +62,13 @@ If you are going to use WaterTAP's functionality, but *do not* plan to contribut
 
 .. _install-idaes-ext:
 
-Installing solvers distributed through IDAES Extensions
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Installing solvers
+^^^^^^^^^^^^^^^^^^
 
-After installing WaterTAP, the ``idaes get-extensions`` command can be used to automatically install the solvers distributed as part of the IDAES Extensions.
+Windows and Linux Users: solvers distributed through IDAES Extensions
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-.. important:: Depending on your operating system, additional steps might be needed. For more information, refer to the `IDAES installation guide <https://idaes-pse.readthedocs.io/en/stable/getting_started/index.html>`_.
```

Comment:
Should we consider keeping this line? AFAIK the need for additional steps before `idaes get-extensions` (e.g. on Linux, installing some system packages) still applies.

Reply:
I agree -- I put it back in.